amr_core:
  ros__parameters:
    wheel_radius: 0.114
    distance_wheels: 0.404
    gear_ratio: 0.03

    can_device: can0
    id_rpm: 1
    id_enc: 1
    Main1: 0x1A1B101
    Main2: 0x1A1B102
    Sub: 0x1B1A101
    Motordriver: 0x00B7B801
    Conveyor: 0x1A1B103
    Sub_Conveyor: 0x1B1A103

    enc_vel_gain: 0.9
    enc_wvel_gain: 0.95
    read_rate: 2.0

    loop_hz: 20.0
    vx_acc: 0.7
    vy_acc: 0.7
    w_acc: 0.7